BackofficeBundle/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -190,3 +191,11 @@
         {% endif %}
     {% endspaceless %}
 {% endblock oo_date_picker_widget %}
+{% block form_legend %}
+    <div class="form-group form-legend">
```

Comment:
class form-legend is never used why add it ? and i think than would be a good idea to use the class help-block of bootstrap